packages/myst-cli/src/transforms/images.ts

```diff
@@ -500,6 +500,40 @@ export async function transformThumbnail(
   }
 }
 
+export async function transformBanner(
```

Comment:
Maybe some deduplication we can do between this and thumbnails, but for now, looks great!

Reply:
Yeah, agree. :)